.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 # Keep tool versions in sync with the versions in requirements-dev.txt
 default_language_version:
     python: python3
+default_stages: [pre-commit]
```

Comment:
Don't know about this one, is it needed? If you replace it with `default_install_hook_types: [pre-commit, commit-msg]` the `commit-msg` hook is also installed by default.

Reply:
`default_install_hook_types: [pre-commit, commit-msg]` is definitely an improvement so that users won't need to install `commit-msg` hook separately, added that 👍 

However, I think `default_stages` is still needed. Without it, many of the hooks are run twice (on pre-commit and commit-msg stages) when committing. That doesn't actually break anything, but it makes pre-commit's output a bit confusing:

```
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...............................................................Passed
check toml...........................................(no files to check)Skipped
check for added large files..............................................Passed
black....................................................................Passed
flake8...................................................................Passed
isort....................................................................Passed
check yaml...........................................(no files to check)Skipped
check toml...........................................(no files to check)Skipped
black................................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
isort................................................(no files to check)Skipped
commitlint...............................................................Passed
```

versus 
```
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...............................................................Passed
check toml...........................................(no files to check)Skipped
check for added large files..............................................Passed
black....................................................................Passed
flake8...................................................................Passed
isort....................................................................Passed
commitlint...............................................................Passed
```

Reply:
Indeed, didn't really notice that behavior earlier. `default_stages` looks good. :+1: